templates/base_index.html

```diff
@@ -63,6 +63,7 @@ <h2>Modern enterprise open&nbsp;source</h2>
     <hr class="p-separator" />
   </div>
 
+  {# hide for now
```

Comment:
Can we remove code commented out code and use version control to retrieve it when we need too?

Reply:
@anthonydillon I guess we will disagree about this forever, but I have removed it for you.